packages/documentation/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -204,7 +160,7 @@ function renderTextarea(args: Args, context: StoryContext) {
       id=${context.id}
       class=${classes}
       defaultValue=${args.value ?? nothing}
-      placeholder=${useAriaLabel ? args.label : ''}
+      placeholder=${useAriaLabel ? args.label : 'My placeholder'}
```

Comment:
I know this was already there before but using labels as placeholders is a bad practice so I would remove this logic: 
```suggestion
      placeholder="My placeholder"
```